metrics/windows/disk.go

```
@@ -73,11 +74,8 @@ func NewDiskGenerator(interval time.Duration) (*DiskGenerator, error) {
 		}
 	}
 
-	r, _, err = PdhCollectQueryData.Call(uintptr(g.query))
-	if r != 0 && err != nil {
-		diskLogger.Criticalf(err.Error())
-		return nil, err
-	}
+	windows.PdhCollectQueryData.Call(uintptr(g.query))
```

Is it okay to discard errors on PdhCollectQueryData?